django_project/feti/views/api.py

```diff
@@ -363,6 +365,16 @@ def filter_model(self, user, options=None, query=None):
         administrative = None
         radius = 0
 
+        # hack for PDF rendering
```

Comment:
please remove this comment lines
For the process description, we could put it in function docs